_layouts/event.html

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+---
+
+{% assign event_id = page.event_id %}
+{% assign event = site.data[event_id] %}
+{% assign details = event.details %}
+{% assign presenters = event.presenters %}
```

Comment:
@aebelenkiy ok so I made a bunch of progress on the template.

This is still not finished, but I think this should give you a good idea of how to do things.

The page now has an `event_id:` in the front matter, which we then use to map to `_data/retreat2025/...` where we have:

- `_data/retreat2025/details.yml`
- `_data/retreat2025/presenters.yml`